mithril-common/src/test_utils/precomputed_keskey.rs

```diff
@@ -0,0 +1,472 @@
+                    220, 220, 162, 147, 131, 31, 196, 79, 81, 155, 147, 174, 207, 163, 31, 244, 85,
+                    71, 88, 193, 11, 61, 194, 232, 63, 74, 21, 243, 216, 70, 159, 59, 11, 99, 25,
+                    60, 127, 75, 203, 165, 56, 127, 155, 221, 154, 96, 209, 23, 64, 49, 207, 111,
+                    154, 194, 184, 85, 48, 24, 169, 68, 61, 239, 41, 195, 100, 139, 184, 175, 181,
+                    109, 61, 124, 214, 82, 4, 99, 171, 180, 101, 78, 131, 84, 74, 164, 225, 0, 14,
+                    115, 131, 233, 124, 69, 61, 54, 25, 50, 251, 15, 184, 187, 141, 222, 201, 97,
+                    167, 113, 47, 94, 117, 164, 207, 155, 101, 247, 46, 24, 60, 199, 144, 196, 209,
+                    253, 159, 1, 192, 159, 97, 32, 8, 147, 157, 216, 54, 123, 35, 74, 244, 26, 138,
+                    27, 153, 5, 89, 112, 243, 169, 21, 68, 99, 44, 179, 205, 161, 90, 220, 12, 141,
+                    37, 234, 63, 159, 247, 62, 225, 183, 212, 207, 199, 176, 90, 8, 187, 162, 124,
+                    118, 100, 132, 60, 34, 220, 226, 214, 95, 134, 59, 13, 182, 175, 213, 82, 25,
+                    95, 251, 109, 249, 188, 69, 137, 131, 202, 171, 23, 214, 26, 173, 144, 170,
+                    222, 204, 14, 109, 141, 19, 216, 165, 244, 212, 129, 243, 70, 219, 221, 82, 44,
+                    7, 82, 197, 185, 19, 18, 68, 168, 228, 93, 67, 89, 184, 156, 88, 18, 35, 238,
+                    243, 44, 124, 224, 104, 9, 226, 184, 36, 27, 246, 152, 34, 109, 17, 236, 33,
+                    64, 51, 196, 247, 88, 32, 119, 241, 75, 110, 133, 126, 118, 77, 10, 87, 179,
+                    214, 97, 160, 87, 144, 82, 170, 30, 43, 155, 167, 35, 98, 250, 69, 187, 236,
+                    227, 230, 98, 0, 1, 252, 121, 245, 69, 52, 222, 191, 81, 227, 62, 94, 55, 209,
+                    128, 116, 150, 60, 88, 173, 5, 15, 223, 47, 156, 220, 127, 121, 48, 184, 189,
+                    192, 28, 250, 168, 141, 197, 234, 152, 186, 243, 83, 116, 22, 49, 85, 174, 172,
+                    200, 91, 82, 77, 64, 159, 212, 52, 176, 244, 163, 3, 71, 26, 208, 7, 203, 116,
+                    175, 70, 241, 93, 21, 186, 169, 164, 146, 54, 81, 52, 172, 22, 240, 34, 69, 46,
+                    81, 72, 101, 224, 174, 156, 201, 235, 225, 86, 1, 108, 140, 16, 74, 98, 155,
+                    127, 30, 251, 200, 245, 237, 39, 120, 2, 187, 34, 66, 143, 201, 193, 231, 229,
+                    145, 21, 28, 10, 252, 195, 10, 49, 182, 141, 29, 78, 105, 35, 153, 223, 75,
+                    104, 204, 182, 125, 113, 0, 0, 0, 0,
+                ],
+                [
+                    110, 122, 177, 245, 90, 69, 146, 26, 145, 133, 25, 58, 88, 167, 119, 56, 97,
+                    116, 176, 133, 208, 27, 58, 66, 60, 200, 2, 230, 0, 168, 108, 14,
+                ],
+            )
+        }
+        _ => return None,
+    };
+    let kes_verification_key = PublicKey::from_bytes(&kes_verification_key_cached).unwrap();
```

Comment:
I would prefer to put this part of code into `fixture_builder` and return raw data ([u8:612] , [u8; 32]) from this file.
What do you think ?